.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -18,10 +18,25 @@ pool: sonicbld
 
 stages:
 - stage: Build
+  variables:
+    CACHE_MODE: none
+    VERSION_CONTROL_OPTIONS: 'SONIC_VERSION_CONTROL_COMPONENTS='
```

Comment:
>SONIC_VERSION_CONTROL_COMPONENTS= [](http://example.com/codeflow?start=30&length=33)

Do you miss the components after `=` ?

Reply:
not missing components. When we generate version files, we need unset this ENV.

Reply:
when we want to control third-party resource version, we need to set this ENV to pip,wget,debian, and so on.

Reply:
I am confused. Do you want to `SONIC_VERSION_CONTROL_COMPONENTS=pip,wget,debian`.
If not, where is it set?

Reply:
Understand. In this PR, we only "generate version files", not "control third-party resource version". Thanks!